### PR TITLE
Removes the ability for botanists to generate kudzu on-demand

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Removes the ability for botanists to generate kudzu on-demand

## Why It's Good For The Game

Botanist players are able to reliably make an event level danger to the station via using random weed generation to forcefully create kudzu. This sucks for the game because when the kudzu gets out of control, either because the botanist rolled midround antag halfway through making it or because they suck at containing kudzu and now the round is a kudzu round, especially if it's a multi-z map where kudzu spread becomes near impossible to contain, like Ice Box or North Star.

Botanists really shouldn't be able to mass produce kudzu on demand, beneficial or not. This was fine when we had players with self control and admins who enforced that self control. Nowadays, people who are spamming kudzu for their third round in a row as a non-antag are protected by the admins who tell people who ahelp the kudzu spammer to "let them cook"

direct quote btw from an admin during a round where a person, after planting their kudzu in a high traffic area, got so upset people kept walking into their kudzu farm by accident that they just planted a new patch of kudzu and left it to grow without any maintenance

Admins are once again unwilling to moderate player behavior so we have to moderate player behavior from the codebase using the hammer instead of the chisel.

## Changelog
:cl:
del: Removes the ability for botanists to generate kudzu on-demand
/:cl:

### CODE REVIEW

**Comment by ShibaInuLord** [comment]
ragepr moment

**Comment by bluetshirtguy** [comment]
Will you please consider adding Kudzu as a potential traitor item for botanists?

**Comment by Iamgoofball** [comment]
@bluetshirtguy thats fine by me but prolly for a different PR

**Comment by GuillaumePrata** [comment]
If botanists can't make Kodzu we can make the random Kodzu event more powerful.
Everyone wins outside of spammer botanists.

**Comment by Mothblocks** [comment]
> Admins are once again unwilling to moderate player behavior so we have to moderate player behavior from the codebase using the hammer instead of the chisel.

nope

**Comment by datorangebottle** [comment]
I don't even like "good" kudzu in the first place, but trying to remove it as a knee-jerk response to drama you started isn't the way to do it, chief.
Anyone wanting to know the context of this PR:
Seth was whining about someone fucking with his kudzu, Goofball barged into #tgstation-general on the discord and started throwing a tantrum about the situation, then kept tantrumming until someone timed him out for five minutes, during which he started on this PR.
TLDR: It's just Goof throwing another tantrum and trying to make the codebase objectively worse for it.

**Comment by SethLafuente** [comment]
All of this stemmed goof ball pissing his pants over a round and an encounter that they weren't involved in naturally a horrible PR botanist mains do not approve

**Comment by Iamgoofball** [comment]
@ninjanomnom @optimumtact can we remove kudzu from botanists yet

**Comment by Misdoubtful** [comment]
Let botanist players cook.

**Comment by nukularpower** [comment]
have botanists not already been nerfed enough?  I've kudzu blasted the station 3 times myself, and not a single one of them was as a botanist, but using kudzu seed packs i randomly found and bred

**Comment by moocowswag** [comment]
I honestly believe kudzu isnt spammed enough if the counters to it are yet to be used or established in the meta.

goof in the future please try to either discover or add your own counters to something rather than jumping to remove a feature, there are tool readily available to the crew that specifically counter kudzu, including the scythes from the holodeck.
